test/gtest/uct/test_amo_and_or.cc

```diff
@@ -0,0 +1,118 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2018.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#include "test_amo.h"
+
+
+class uct_amo_and_or_test : public uct_amo_test {
```

Comment:
why separate to add/xor and and/or? can they be in same class?
i think the separation should be op/fop

Reply:
because add/xor are cyclic operators - it doesn't seek to any constant value, but this is not true for and/or - for example operation ```rand | rand | rand ...``` may provide result MAX_VALUE, that is why I used bit different tests